### PR TITLE
Add variable undeclared ($url)

### PR DESCRIPTION
**Pull request by ColonelMoutarde**
Closes #

Changes proposed in this pull request:
Add variable undeclared ($url)

How to test the feature manually:

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]
Thanks :+1:

**Review by aledeg — CHANGES_REQUESTED** [review]
I think it is better to change the error message instead of adding a useless parameter to the method

**Comment by ColonelMoutarde** [comment]
As you want :)

File: app/Exceptions/ZipException.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
 class FreshRSS_Zip_Exception extends Exception {
 	private $zipErrorCode = 0;
 
-	public function __construct($zipErrorCode) {
+	public function __construct($zipErrorCode, $url = null) {
 		parent::__construct('ZIP error! ' . $url, 2141);
```
Comment (Alkarex) [inline app/Exceptions/ZipException.php]:
```suggestion
	public function __construct($zipErrorCode) {
		parent::__construct('ZIP error!', 2141);
```

I believe this is what @aledeg suggests. You can just click on accept here @ColonelMoutarde

**Review by aledeg — APPROVED** [review]

**Pull request merged**